From 9f90ef945382ef77e00b90ee0d0137632e3a77c9 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 15 May 2020 13:50:00 +0300
Subject: [PATCH] [SW-11875] goya2: modify default syscfg value for onix

This patch modifies default syscfg efuse value to enable signature
validation and disable recovery for onix platform.

Change-Id: I47274665aa4fe77a9006f9465e927c60fd02fa60
---

diff --git a/goya2_arc/onix.c b/goya2_arc/onix.c
index 9cd7979..50aaae6 100644
--- a/goya2_arc/onix.c
+++ b/goya2_arc/onix.c
@@ -12,7 +12,9 @@
 #define EFUSE_SYSCFG_ONIX_DEFAULT \
 				(EFUSE_SYSCFG_FW_VERSION_LOCATION_SPI | \
 				 EFUSE_SYSCFG_KEY_LOCATION_SPI | \
-				 EFUSE_SYSCFG_REVOKATION_LOCATION_SPI)
+				 EFUSE_SYSCFG_REVOKATION_LOCATION_SPI | \
+				 EFUSE_SYSCFG_ENABLE_VALIDATION_MASK | \
+				 EFUSE_SYSCFG_GPIO02_RECOVERY_CONTROL)
 
 int fuse_read(uint32_t word, uint32_t *val)
 {
